pkg/crc/api/types.go

```diff
@@ -6,23 +6,10 @@ import (
 )
 
 type Server struct {
-	handler  RequestHandler
+	handler  *Handler
 	listener net.Listener
 }
 
-type RequestHandler interface {
-	Start(json.RawMessage) string
-	Stop() string
-	Status() string
-	Delete() string
-	GetVersion() string
-	SetConfig(json.RawMessage) string
-	UnsetConfig(json.RawMessage) string
-	GetConfig(json.RawMessage) string
-	GetWebconsoleInfo() string
-	Logs() string
-}
-
```

Comment:
Not directly related to this commit I believe?

Reply:
extracted to a specific commit